assets/js/generator.js

```diff
@@ -158,31 +158,30 @@ function generate() {
         currentGeneratorRequest.abort();
     }
 
-    currentGeneratorRequest = $.jsonp({
-        url: config.APY_URL + '/generate',
-        beforeSend: ajaxSend,
-        complete: function () {
-            ajaxComplete();
-            currentGeneratorRequest = undefined;
-        },
-        data: {
-            'lang': generatorMode,
-            'q': $('#morphGeneratorInput').val()
-        },
-        success: function (data) {
-            $('#morphGenOutput').empty();
-            for(var i = 0; i < data.length; i++) {
-                var div = $('<div data-toggle="tooltip" data-placement="auto" data-html="true"></div>');
-                var strong = $('<strong></strong>').text(data[i][1].trim());
-                var span = $('<span></span>').html('&nbsp;&nbsp;&#8620;&nbsp;&nbsp;' + data[i][0]);
-                div.append(strong).append(span);
-                $('#morphGenOutput').append(div);
-            }
-            $('#morphGenOutput').removeClass('blurred');
-        },
-        error: function (xOptions, error) {
-            $('#morphGenOutput').text(error);
-            $('#morphGenOutput').removeClass('blurred');
-        }
-    });
+    var request = {'lang': generatorMode};
+    request['q'] = $('#morphGeneratorInput').val();
```

Comment:
`request['q']` will probably still error in favor of `.q`. Don't worry too much about these kinds of errors if you can't fix them. We can always override the linter (you can Google for how).